src/pages/groupComparison/CopyNumberEnrichments.tsx

```diff
@@ -20,11 +20,11 @@ export interface ICopyNumberEnrichmentsProps {
 @observer
 export default class CopyNumberEnrichments extends React.Component<ICopyNumberEnrichmentsProps, {}> {
     @autobind
-    private onChangeProfile(m:MolecularProfile) {
-        this.props.store.setCopyNumberEnrichmentProfile(m);
+    private onChangeProfile(profileMap:{[modecularProfileId:string]:MolecularProfile}) {
```

Comment:
looks like in the code its actually studyId to molecular profile right? if so then can you rename the parameter studyId, here and elsewhere